acclimatise/model.py

```diff
@@ -293,15 +293,21 @@ def _name_from_arg(self) -> typing.Iterable[str]:
 
     def get_type(self) -> cli_types.CliType:
         # Try the argument name, then the flag name, then the description in that order
-        arg_type = self.args.get_type()
-        if arg_type is not None:
-            return arg_type
-
-        flag_type = infer_type(self.full_name())
-        if flag_type is not None:
-            return flag_type
-
-        return infer_type(self.description) or cli_types.CliString()
+        tpe = None
+        tpe_cand = [
```

Comment:
I prefer descriptive variable names where possible. `typ` and `type_candidates` please.

Reply:
> I would have though your example in #37 would predict None, then CliFile() then CliFile() respectively, and then return CliFile(). We only use subclasses for CliFileSystemType, and only recently, but the inference will never return CliFileSystemType, so it seems irrelevant.

True. I already forgot that we implemented the distinction between in/output files/dirs by properties and not subclasses.Then I will check for `isinstance`.

Reply:
Ah that makes more sense